### DISCUSSION TITLE
Importing pois in the GPX format has broken

### DISCUSSION THREAD

**Discussion by GpxFeed** [devemux86/cruiser#264] [Q&A] [answered]
Previous versions have shown both data types - the "wpt" part and the "trk" part of an OSM poi.

On importing a gpx-file as overlay the "trk" part now has disappeared.

Wiki
- https://de.wikipedia.org/wiki/GPS_Exchange_Format

Examples
- https://github.com/GpxFeed/campgrounds

Fortunately, we still had a previous desktop version available on my netbook on our Burgundy tour.

Bug or feature?

BTW
Still not realized is the "link" tag. You can have a look at this at GPXSee.

-- 
Brinki

**Comment by devemux86** [comment, marked as answer]
@GpxFeed thanks for the report.

I am not sure I understand clearly what you mean? Is it about desktop Cruiser?

For example, I tested the [andorra-fuel.gpx](https://github.com/GpxFeed/campgrounds/blob/master/gpx/andorra-fuel.gpx). It contains waypoints and multiple tracks.

- If we import it as Favorites, the waypoints are only imported.
- If we import it as Route overlay, then the waypoints + tracks are imported.

Like in the image, cyan waypoints and blue tracks:

![Cruiser](https://github.com/devemux86/cruiser/assets/3484020/572d4561-ca2a-481d-816c-eee064273eec)

**Reply by GpxFeed** [reply]
> Is it about desktop Cruiser?

Yes, 

- Desktop Cruiser on Open Suse Tumbleweed, Kernel 6.3.1-2.
- Cruiser 3.0.24 and 3.0.28

I selected the following:

- Routes
-  Import
-   andorra-campsites.gpx
-    Overlay [OK]

See attached hardcopy.

On the way we had the same effect with the Android version on a Nokia and only part of the imported pois became visible - when zooming in.

![andorra](https://github.com/devemux86/cruiser/assets/66617950/38f975a3-6bbc-4577-8068-85c604caba1f)

**Reply by devemux86** [reply]
@GpxFeed when I select "Routes | Import | Overlay" (see image) I see all waypoints and tracks imported.

What do you mean "only part of the imported pois became visible"?
What exactly is missing?

![Cruiser](https://github.com/devemux86/cruiser/assets/3484020/e5c046e4-d353-40ae-beb2-1008c05eb86b)

**Reply by devemux86** [reply]
Also why are you trying to import / overlay POI with route menu?
This is more to import tracks or waypoints for route planning.

You can import them as favorites and use them everywhere.

**Reply by GpxFeed** [reply]
Sorry, it's jinxed - at the moment it works again - at least on the desktop. Can't reproduce it anymore. On the Desktop it was probably due to a color setting.

I'll have to investigate this on Android in the future.

> What do you mean "only part of the imported pois became visible"?
> What exactly is missing?

On Android some of the imported pois where missing on zooming in - even to highest resolution. Is there a cross-check with already existing data?

> You can import them as favorites and use them everywhere.

Yes, but also interesting information about the poi's surroundings will get lost.

**Reply by devemux86** [reply]
Perhaps you mean the marker clusters to avoid problems in performance with too many markers.
In Android Cruiser it can be controlled in "Settings | Map | Appearance | Marker groups (zoom)".

**Reply by 0709wiwiwi** [reply]
@GpxFeed 
Desired are clickable "links" ?
Easily achievable ? No idea.
In gpx 1.1 find in the "link" tag.
Can even be present in "desc"
Example gpx file. 
[BOM_navtrkpt_link_wpt.gpx.txt](https://github.com/devemux86/cruiser/files/12238659/BOM_navtrkpt_link_wpt.gpx.txt)

And in the further future ? Possibly more can be done.
A gpz file inclusive images importable in Locus or the gpx viewer app.
[BOM_navtrk.gpz.txt](https://github.com/devemux86/cruiser/files/12238677/BOM_navtrk.gpz.txt)

**Reply by devemux86** [reply]
> Desired are clickable "links" ?
Easily achievable ? No idea.

This is indeed not easy.

Please let's discuss this in its topic:
- https://github.com/devemux86/cruiser/discussions/55
